openedx/openedx-learning
# feat: Add export_id field on Taxonomy

**Description**

This PR adds a new `export_id` field on `Taxonomy` model.

**Support information**

- Github issue: https://github.com/openedx/modular-learning/issues/183
- Internal Ticket: [FAL-3620](https://tasks.opencraft.com/browse/FAL-3620)

**Testing instruction**

- Verify that all tests pass and that they cover the requirements.

Before run migration

- Start a server with `python manage.py runsever`.
- Create a superuser with `python manage.py createsuperuser`.
- Go to [Taxonomy list](http://127.0.0.1:8000/admin/oel_tagging/taxonomy/) on Django Admin.
- Create two Taxonomies with the same name.
- Run the migrations: `python manage.py migrate`.
- Verify that the migrations have been executen without errors.
- On Django admin create a new Taxonomy and verify that you need to add an `export_id`
- Verify that you can edit the `export_id` on Django admin.
- Verify the validations of the `export_id`
